.github/workflows/mvnw.yml

```diff
@@ -4,10 +4,10 @@ on: [push]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     strategy:
       matrix:
-        java: [ '11', '17' ]
+        java: [ '17.0.4+8' ]
```

Comment:
Is it intentional this is different that the Circle CI version (17.0.3)?

Reply:
No, I think this is just pinned to the same as in the webservice repo.